c11threads.h

```diff
@@ -37,11 +77,34 @@ Main project site: https://github.com/jtsiomb/c11threads
 #endif
 
 /* types */
+#if defined(_WIN32) && !defined(C11THREADS_PTHREAD_WIN32)
+typedef unsigned long thrd_t;
+#pragma pack(push, 8)
```

Comment:
Is the forced alignment necessary? Remember that MSVC is not the only win32 compiler, so if we really need to force alignment, we'd have to also make it work for gcc/mingw at least. That could be a spearate commit later on, I could even add it myself at some point and test with a cross-compiler, but if it can be avoided altogether, it would be better.

Reply:
I had copied that from the declaration, but it really isn't necessary. I'll remove it.